src/ScalarPotentials/ElectricPotential.jl

```diff
@@ -40,4 +40,20 @@ function ElectricPotential(nt::NamedTuple)
     N = get_number_of_dimensions(grid)
     ElectricPotential{T, N, S, typeof(grid.axes)}( ustrip.(uconvert.(u"V", nt.values)), grid)
 end
-Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
\ No newline at end of file
+Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
+
+"""
+    get_ticks_at_positions_of_large_gradient(field::ElectricPotential)
+
+The electric potential is analyzed in order to find and return ticks where the gradient (electric field) is strong.
+"""
+function get_ticks_at_positions_of_large_gradient(field::ElectricPotential{T, 3}) where T

```

Comment:
Would it make sense to make this function for arguments of type `ScalarPotential`?
And can you rename the `field` variable to `epot` (as we try to use consistent naming for variables of the same type)?